doc/make.jl

```diff
@@ -38,8 +38,16 @@ for dir in readdir(STDLIB_DIR)
     end
 end
 
+# Generate a suitable markdown file from NEWS.md and put it in src
+str = read(joinpath(@__DIR__, "..", "NEWS.md"), String)
+splitted = split(str, "<!--- generated by NEWS-update.jl: -->")
```

Comment:
Should we also split out the previous versions release notes that might not have been removed from NEWS.md yet?

Reply:
I think it is fine to have them there for "latest" versions of the manual. For stable the old stuff will have been removed.